test/test_queue.js

```diff
@@ -1979,6 +1979,46 @@ describe('Queue', function() {
       });
     });
 
+    it('should not retry a job if the custom backoff returns -1', function(done) {
+      queue = utils.buildQueue('test retries and backoffs', {
+        settings: {
+          backoffStrategies: {
+            custom: function() {
+              return -1;
+            }
+          }
+        }
+      });
+      var tries = 0;
+      queue.isReady().then(function() {
```

Comment:
I think it should not be needed to check for readyness here.

Reply:
Okay, I'll remove the isReady() call